### PR TITLE
Switch to absolute import paths and update docs

### PR DESCRIPTION
**Pull request by kev-inn**

### CODE REVIEW

File: src/main/python/docs/source/matrix.rst [hunk 1]
```diff
@@ -14,17 +14,23 @@
...
 ==========
 
 OperationNode
 -------------
 
+  .. todo
+    The explanation for overloade methods seems weird and does not really describe which
+    methods we mean (magic methods for operators like `+`, `*` etc.).
+    Also I don't understand why that would mean that they return an ``OpeartionNode``.
+
```
Comment (kev-inn) [inline src/main/python/docs/source/matrix.rst:28]: Some discussion on this would be great @Baunsgaard

Comment (Baunsgaard) [reply src/main/python/docs/source/matrix.rst]: Probably @sebwrede who commented on this.

File: src/main/python/systemds/matrix/matrix.py [hunk 2]
```diff
@@ -22,9 +22,9 @@
 import numpy as np
 from py4j.java_gateway import JVMView, JavaObject
 
-from ..utils.converters import numpy_to_matrix_block
-from ..script_building.dag import VALID_INPUT_TYPES
-from .operation_node import OperationNode
+from systemds.utils.converters import numpy_to_matrix_block
+from systemds.script_building.dag import VALID_INPUT_TYPES
+from systemds.matrix.operation_node import OperationNode
```
Comment (Baunsgaard) [inline src/main/python/systemds/matrix/matrix.py:27]: :+1:

File: src/main/python/docs/source/matrix.rst (hunk 1 above)
Comment (Baunsgaard) [reply src/main/python/docs/source/matrix.rst:28]: well call me :telephone_receiver:

File: src/main/python/systemds/matrix/operation_node.py [hunk 3]
```diff
@@ -16,13 +16,13 @@
...
-from ..utils.helpers import get_gateway, create_params_string
-from ..utils.converters import matrix_block_to_numpy
-from ..script_building.script import DMLScript
-from ..script_building.dag import OutputType, DAGNode, VALID_INPUT_TYPES
 from typing import Union, Optional, Iterable, Dict, Sequence
 
+from systemds.utils.helpers import get_gateway, create_params_string
+from systemds.utils.converters import matrix_block_to_numpy
+from systemds.script_building.script import DMLScript
+from systemds.script_building.dag import OutputType, DAGNode, VALID_INPUT_TYPES
```
Comment (Baunsgaard) [inline src/main/python/systemds/matrix/operation_node.py:24]: :+1:

File: src/main/python/systemds/script_building/script.py [hunk 4]
```diff
@@ -14,13 +14,14 @@
...
+from typing import Any, Collection, KeysView
 
 from py4j.java_collections import JavaArray
 from py4j.java_gateway import JavaObject
 
-from ..utils.helpers import get_gateway
-from ..script_building.dag import DAGNode, VALID_INPUT_TYPES
+from systemds.utils.helpers import get_gateway
+from systemds.script_building.dag import DAGNode, VALID_INPUT_TYPES
+from typing import Union, Optional, Dict
```
Comment (Baunsgaard) [inline src/main/python/systemds/script_building/script.py:24]: :+1:

**Review by Baunsgaard — COMMENTED** [review]
LGTM

**Review by corepointer — APPROVED** [review]
LGTM, thanks, will merge.

File: src/main/python/docs/source/matrix.rst (hunk 1 above)
Comment (mboehm7) [reply src/main/python/docs/source/matrix.rst:28]: call you on the red emergency phone for PR comments - nice, good to know ;-).

**Pull request closed without merging**